src/Tizen.System.SystemSettings/Tizen.System.SystemSettings/SystemSettingsEventArgs.cs

```diff
@@ -1186,4 +1186,36 @@ public bool Value
             }
         }
     }
+
+    /// <summary>
+    /// EventArgs type for the RotaryEventEnabledChanged event.
+    /// </summary>
+    /// <privilege>http://tizen.org/privilege/systemsettings.admin</privilege>
+    /// <privlevel>platform</privlevel>
+    /// <feature>http://tizen.org/feature/systemsetting</feature>
+    /// <exception cref="ArgumentException">Invalid Argument</exception>
+    /// <exception cref="NotSupportedException">Not Supported feature</exception>
+    /// <exception cref="InvalidOperationException">Invalid operation</exception>
+    /// <exception cref="UnauthorizedAccessException">Thrown when application does not have privilege to access this method.</exception>
+    /// <since_tizen> 6 </since_tizen>
+    public class RotaryEventEnabledChangedEventArgs : EventArgs
+    {
+        private readonly bool _rotaryEventEnabled;
+        internal RotaryEventEnabledChangedEventArgs(bool val)
+        {
+            _rotaryEventEnabled = val;
+        }
+
+        /// <summary>
+        /// Indicates whether developer option state is enabled on the device or not.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public bool Value
```

Comment:
You can use default implemented property. You can update value on constructor even if it has only getter
``` c#
 public bool Value { get; }
```